packages/shared/components/Route.svelte

```diff
@@ -1,9 +1,11 @@
 <script lang="ts">
-    import { appRoute, AppRoute } from '@core/router'
+    import { AppRoute } from '@core/router'
+    import { Writable } from 'svelte/store'
 
     export let route: AppRoute
+    export let routeStore: Writable<AppRoute>
 </script>
 
-{#if $appRoute === route}
+{#if $routeStore === route}
```

Comment:
this component is so simple, that im thinking if we actually need it, we dont use it anywhere else because we manage the routing directly in the routing views, so Im thinking we might be able to get rid of this component altogether. We had it before because we used to have animations between routing transitions, but now i dont know if it adds value, wdyt?

Reply:
Removed the Route component